assignments/prototype-refactor.js

```diff
@@ -7,3 +7,134 @@ Prototype Refactor
 2. Your goal is to refactor all of this code to use ES6 Classes. The console.log() statements should still return what is expected of them.
 
 */
+/*
+=== GameObject ===
+* createdAt
+* name
+* dimensions (These represent the character's size in the video game)
+* destroy() // prototype method that returns: `${this.name} was removed from the game.`
+*/
+
+class GameObject {
+    constructor(attributes) {
+        this.createAt = attributes.createdAt; 
```

Comment:
Please check the spelling of your constructor variables to prevent future console errors.